build/bin/smoke-onos-e2t

```diff
@@ -69,7 +69,7 @@ $cli_command "$onos_test/build/bin/check-topo-relations" "topo get relations"
 
 $cli_command "$onos_test/build/bin/check-topo-entities" "topo get entities"
 
-$cli_command "$onos_test/build/bin/check-e2t-connections" "e2t list connections"
+$cli_command "$onos_test/build/bin/check-e2t-connections" "topo get entities --kind e2t"
```

Comment:
this should be Control relations no? list of connections are correspondent to list of Control relations

Reply:
I know... I was planning to check E2T nodes and E2T connections, but had to hold off due to current E2T issues.

Reply:
I think I will just do what you suggested, I was going to correlate the results of e2t entities and relations, but just realized that the retry is external to this